nncf/experimental/common/graph/netron.py

```diff
@@ -159,14 +160,17 @@ def get_graph_desc(graph: NNCFGraph,
                 )
             )
 
+        if get_attributes_fn is None:
```

Comment:
I think we should put this at the beginning of the `save_for_netron()` method implementation. I mean
```python
def save_for_netron(...):
    if get_attributes_fn is None:
        get_attributes_fn = lambda x: {
            'metatype': str(x.metatype.name),
        }
```

Reply:
Done